apps/components/impact/latam-impact-section.tsx

```
@@ -0,0 +1,114 @@
+          </Heading>
+          <Text textAlign="center" color="$gray600" mt={8}>
+            KindFi’s Stellar-based escrow system ensures every contribution is secure, transparent, and
+            impactful, with milestone-based fund releases.
+          </Text>
+        </MotiView>
+
+        {/* Feature Cards */}
+        <HStack space="md" justifyContent="space-between" flexWrap="wrap">
+          {featureCards.map((card, index) => (
+            <MotiView
+              key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)